src/Management/src/Tracing/Steeltoe.Management.Tracing.csproj

```diff
@@ -2,13 +2,13 @@
   <PropertyGroup>
     <TargetFrameworks>net8.0</TargetFrameworks>
     <Description>Enables request tracing in distributed systems.</Description>
-    <PackageTags>management;monitoring;distributed tracing</PackageTags>
+    <PackageTags>management;monitoring;distributed tracing;</PackageTags>
```

Comment:
```suggestion
    <PackageTags>management;monitoring;distributed tracing;go-router</PackageTags>
```

Reply:
This was actually not directly connected to the go-router, rather it implements the same trace log format found in Spring Cloud Sleuth, but I am skeptical that anybody searching for either of those terms would actually want this package